docs/DISTRIBUTION.md

```diff
@@ -113,6 +113,11 @@ date and hour directory also contain an index file each, listing all the subdire
 concept, but writing this structure to the local disk is not supported. Therefore, this country file will be assembled as
 `version/v1/diagnosis-keys/country/index`. The `/index` part will then be removed upon S3 upload.
 
+The `origin_country` (ISO-3166 country code) parameter defines which country folder is being created.
```

Comment:
Suggested rewording:
"...parameter refers to the country in which the CWA clients and backend operate. It is used to create a folder on the CDN distribution paths, where so called "national key packages" are distributed. In essence, these packages  contain keys submitted by CWA clients as well as keys from visitors of the country, that have been downloaded from the Federation Gateway Service."